docs/proposals/local-cluster-ux.md

```diff
@@ -0,0 +1,184 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Kubernetes Local Cluster Experience
+
+This proposal attempts to improve the existing local cluster experience for kubernetes.
+The current local cluster experience is sub-par and often not functional.
+This proposal is a RFC!
+
+## Primary Goals
+
+From a high level the goal is to make it easy for a new user to run a Kubernetes cluster and play with curated examples that require least amount of knowledge about Kubernetes.
+These examples will only use kubectl and only a subset of Kubernetes features that are available will be exposed.
```

Comment:
A few that I can think off the top of my head are,

- L3 & L7 loadbalancing
- Most of the volume plugins
- Horizontal Pod autoscaling - this needs heapster addon

I'm sure we will find more along the way.

On Mon, Apr 18, 2016 at 12:11 PM, Lucas Käldström <notifications@github.com>
wrote:

> In docs/proposals/local-cluster-ux.md
> <https://github.com/kubernetes/kubernetes/pull/24106#discussion_r60116777>
> :
>
> > +--
> > +
> > +<!-- END STRIP_FOR_RELEASE -->
> > +
> > +<!-- END MUNGE: UNVERSIONED_WARNING -->
> > +
> > +# Kubernetes Local Cluster Experience
> > +
> > +This proposal attempts to improve the existing local cluster experience for kubernetes.
> > +The current local cluster experience is sub-par and often not functional.
> > +This proposal is a RFC!
> > +
> > +## Primary Goals
> > +
> > +From a high level the goal is to make it easy for a new user to run a Kubernetes cluster and play with curated examples that require least amount of knowledge about Kubernetes.
> > +These examples will only use kubectl and only a subset of Kubernetes features that are available will be exposed.
>
> Which features won't be exposed?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24106/files/4d4288e76854d11c1f2db2ffd002a3ed041995fe#r60116777>
>